mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -125,12 +125,13 @@
 $wgExtraNamespaces[NS_PRIVATE] = "Private";
 
 ## Adding new user group private which is blocking reading and editing pages in private namespace.
-$wgGroupPermissions['private'] = []; 
+$wgGroupPermissions['private'] = [];
 $wgNamespacePermissionLockdown[NS_PRIVATE]['edit'] = [ 'private' ];
 $wgNamespacePermissionLockdown[NS_PRIVATE]['read'] = [ 'private' ];
 
+$wgVisualEditorRebaserURL = 'http://localhost:8081';
```

Comment:
@LizzAlice will this work? I guess in production this might need to point to another URL?